shell/edit/provisioning.cattle.io.cluster/SelectCredential.vue

```diff
@@ -104,7 +104,7 @@ export default {
     options() {
       const out = this.filteredCredentials.map((obj) => {
         return {
-          label: obj.nameDisplay,
+          label: `${ obj.nameDisplay }/${ obj.id }`,
```

Comment:
I don't think we should add the IDs like this - and formatting with a `/` is a bit confusing,

If I was being really clever here, I'd look through the list of credential names first and make a note of the ones that have duplicates.

Then for the label, if the name is one with a duplicate I'd add the id in parenthesis afterwards, otherwise I'd format the ones without duplicates as they currently are (name only).

Reply:
Yeah, was unsure about this, but I thought the best procedure was to add some kind of a distinction between those duplicates on the `LabeledSelect`